fe1-web/src/features/home/screens/ConnectOpenScan.tsx

```diff
@@ -70,7 +70,7 @@ const ConnectOpenScan = () => {
     });
   }, [navigation]);
 
-  const handleScan = (data: string | null) => {
+  const handleScan = async (data: string | null) => {
```

Comment:
since this function is now `async`, do we need to check that the component is still mounted before initiating the navigation ? I know we do to change state, I'm not sure about navigation.

Also, this function is rather huge, we might want to break it up a bit 😅

Reply:
True, otherwise we'll have interesting navigation problems again 😅 (when closing the scanner right after a qr code was visible)